packages/autocomplete-core/propGetters.ts

```diff
@@ -46,7 +46,16 @@ export function getPropGetters<TItem>({
       onSubmit: event => {
         event.preventDefault();
```

Comment:
I'm wondering if preventing the default behavior should be part of the default `onSubmit` implementation. Are there any cases where users would like to use the default `submit` form event?

Reply:
I don't see any use case where user would want to keep default submit event.